.gitignore

```diff
@@ -3,3 +3,4 @@ package-lock.json
 .idea
 coverage/
 *.log
+.DS_Store
```

Comment:
pet peeve of mine, but this should really be in your global `gitignore` since its a platform dependent file

Reply:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

```
git config --global core.excludesfile ~/.gitignore_global
```